packages/coinstac-ui/app/render/components/projects/form-project.js

```diff
@@ -376,7 +385,7 @@ FormProject.propTypes = {
         tags: PropTypes.object.isRequired,
       })
     ).isRequired,
-    metaCovariateMapping: PropTypes.arrayOf(PropTypes.number),
+    metaCovariateMapping: PropTypes.object.isRequired,
```

Comment:
Bad `metaCovariateMapping` shape was causing those React errors in the UI. This change fixes them.